client/src/app.tsx

```diff
@@ -16,21 +16,27 @@ function App() {
   const appId = process.env.HATHORA_APP_ID;
   const token = useAuthToken(appId);
   const [connection, setConnection] = useState<HathoraConnection | undefined>();
-  const [displayMetadata, setDisplayMetadata] = useState<DisplayMetadata>({ serverUrl: "" });
+  const [displayMetadata, setDisplayMetadata] = useState<DisplayMetadata>({ serverUrl: "", winningScore: 15 });
```

Comment:
my intention here is for this to just hold like fields that we display in phaser--if we're adding some actual game logic i'd like to change the name

Reply:
updated to `SessionMetadata`